demo/scripts/installCC.sh

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright IBM Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#!/bin/bash
+
+#  There are 2 installs, followed by (instantiate and upgrade)
+#  with (install + instantiate) executed once, the _correct_ chaincode was not getting installed
+#  when run using scripts.  It may be a delay timing issue.  Yet to figure out what the correct reason is
+#  for this behaviour.  "sleep 15" has been added in an attempt to fix the above mentioned error.
+#  When (install + instantiate) commands are executed by hand, it works.  With scripting, the delay 
+#  due to manual action is absent.  Is that the reason?? 
+#  This version works.  TODO:  To come back and identify the reason for this error
+#  above comment is not valid anymore;  But this version works !  
+set -ev
+export WAIT_TIME=15
+export version=1.1
+
+docker image prune -f
+docker exec cli peer chaincode install -n mockcc -v ${version} --path /opt/chaincode/javascript  -l NODE
```

Comment:
ideally, chaincode name is also read from config.json so we have a single place for configuration.

Reply:
These scripts are not intended to be used in a production setup.   It is presented to be used during dev time.  

This is work in progress.  To read from a json file, currently jq can be used.  However, since jq will be an additional dependency, it is not preferred.  Until a different tool is identified, it is hardcoded here.

Reply:
> These scripts (installCC.sh and upgradeCC.sh) are not intended to be used in a production setup. It is presented in this PR to be used during dev time for integration purposes.
> 
> Also, this is work in progress. To read from a json file, currently jq can be used. However, since jq will be an additional dependency, it is not preferred. Until a different tool is identified, it is hardcoded here.

EDIT: Now that jq is added to the base image, will change this script to pick up chaincode name from config.json

Reply:
(additional reply for marcus) for FPC net you anyway will have to run them on the peer itself and with ${PEER_CMD} (the same of course also would apply to upgradeCC.sh)

Reply:
> ideally, chaincode name is also read from config.json so we have a single place for configuration.

chaincode name is not only input needed for the scripts, installCC.sh and upgradeCC.sh.  (the version number is the one that changes frequently during dev time.   Hence, used as input to upgradeCC.sh)

Reading chaincode name from config.json will be an _incomplete_ work and reading all the inputs (chaincode name, chaincode folder, version, orderer address, channel name, args to chaincode) is not worth the effort now for 2 reasons. 
- it will be an overkill for a convenience script intended for repeated use in dev time. 
-  Integrated scripts which bring up end-to-end demo network will have to do this.  As Michael pointed out, these are to be done using {PEER_CMD} FPC peer wrapper.

So, I propose that I remove these scripts from this PR since these use hardcoded values which is not preferred.

Reply:
yes, I think the scripts folder can be removed from this PR. We will add them later.

Reply:
why remove? Still useful if you would want to use your go mock chaincode where you do not need the fpc network?

Reply:
Given that the mock chaincode is not part of this PR. having the scripts to use with mockcc does not help.  These scripts are available in the branch _which_ has mockcc (and the work is not lost). 

Moreover, the main purpose of this PR is to bring in backend client for the demo with FPC peer. The scripts can be part of another PR which _can_ address the dependencies.

Reply:
Since the purpose of this PR is to merge the backend client code, will remove the scripts which work _only_ with non-FPC / fabric network.  
If the scripts are needed, I would add it another PR with all the hardcoded input and dependencies addressed.